mobly/snippet/client_base.py

```diff
@@ -418,14 +418,8 @@ def handle_callback(self, callback_id, ret_value, rpc_func_name):
       The callback handler object.
     """
 
-  def stop_server(self):
-    """Proxy function of do_stop_server."""
-    self.log.debug('Stopping snippet %s.', self.package)
-    self.do_stop_server()
-    self.log.debug('Snippet %s stopped.', self.package)
-
   @abc.abstractmethod
-  def do_stop_server(self):
+  def stop_server(self):
```

Comment:
should this only kill the server or should this also clean up any socket/connection on the local host?

Reply:
It also kills connection. Actually, it releases all the requested resources. 

So may be better to rename it to `stop` or `kill`?

Reply:
The docstring should clarify this?

Reply:
Modified.